crypto/x509/by_store.c

```diff
@@ -116,7 +116,11 @@ static int by_store_ctrl_ex(X509_LOOKUP *ctx, int cmd, const char *argp,
             argp = ossl_safe_getenv(X509_get_default_cert_dir_env());
 
         if (argp == NULL)
+#ifndef _WIN32
             argp = X509_get_default_cert_dir();
+#else
+            argp = "org.openssl.winstore://";
+#endif
```

Comment:
Wondering why this isn't within `X509_get_default_cert_dir()`?

Reply:
From the man page "The X509_get_default_cert_dir() function returns a default delimeter-separated
list of paths to a directories containing trusted CA certificates named in the hashed format" - so having it return a store URI would be kind of weird.

Reply:
A `X509_get_default_cert_store()` would be more logical.